src/components/movie-view/movie-view.jsx

```diff
@@ -35,7 +33,9 @@ class MovieView extends Component {
             <span className='value'>{movie.Director.Name}</span>
```

Comment:
a improvements you can default to empty string the following way:
`{movie.Director.Name || ''}` because if Director or name is undefined the app is going to crash! the same for all destructing object